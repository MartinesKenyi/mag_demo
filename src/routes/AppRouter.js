import React from 'react'

import {
    BrowserRouter as Router,
    Route,
    Switch,
    Redirect
} from 'react-router-dom';
import { Confirmacion } from '../view/Confirmacion';
import { Datos } from '../view/Datos';
import { Suscripcion } from '../view/Suscripcion';


export const AppRouter = () => {
    return (
        <Router>
            <div>
                <Switch>

                    <Route
                        exact
                        path="/"
                        component={Suscripcion}
                    />
                    <Route
                        exact
                        path="/suscripcion"
                        component={Suscripcion}
                    />
                    <Route
                        exact
                        path="/datos"
                        component={Datos}
                    />

                    <Route
                        exact
                        path="/confirmacion"
                        component={Confirmacion}
                    />

                    <Redirect to="/" />

                </Switch>
            </div>
        </Router>
    )
}
