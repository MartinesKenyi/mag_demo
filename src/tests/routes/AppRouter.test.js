import React from 'react';
import { shallow } from 'enzyme';
import { AppRouter } from '../../routes/AppRouter';

describe('Pruebas en <AppRouter />', () => {
   
    test('Debe de mostrarse correctamente', () => {
        const wrapper = shallow( <AppRouter /> );

        expect(wrapper).toMatchSnapshot();
    });    
});
