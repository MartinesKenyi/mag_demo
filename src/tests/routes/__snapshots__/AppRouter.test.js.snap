// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <AppRouter /> Debe de mostrarse correctamente 1`] = `
<BrowserRouter>
  <div>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/suscripcion"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/datos"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/confirmacion"
      />
      <Redirect
        to="/"
      />
    </Switch>
  </div>
</BrowserRouter>
`;
